apps/src/lib/util/audioApi.js

```diff
@@ -109,6 +115,53 @@ export const commands = {
     } else {
       Sounds.getSingleton().stopAllAudio();
     }
+  },
+  playSpeech(opts) {
+    apiValidateType(opts, 'playSpeech', 'text', opts.text, 'string');
+    apiValidateType(opts, 'playSpeech', 'gender', opts.gender, 'string');
+    const speechConfig = SpeechConfig.fromAuthorizationToken(
+      appOptions.azureSpeechServiceToken,
+      appOptions.azureSpeechServiceRegion
+    );
+    speechConfig.speechSynthesisOutputFormat =
+      SpeechSynthesisOutputFormat.Audio16Khz32KBitRateMonoMp3;
+
+    let voice;
+    if (opts.gender === 'male') {
+      voice = 'en-US-BenjaminRUS';
+    } else {
+      voice = 'en-US-AriaRUS';
+    }
+    const synthesizer = new SpeechSynthesizer(speechConfig, undefined);
+    let ssml = `<speak version="1.0" xmlns="https://www.w3.org/2001/10/synthesis" xml:lang="en-US"><voice name="${voice}">${
+      opts.text
+    }</voice></speak>`;
+    synthesizer.speakSsmlAsync(
+      ssml,
+      result => {
+        let forceHTML5 = false;
```

Comment:
nit: this can be simplified to 
`let forceHTML5 = window.location.protocol === 'file:'`